sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Release History
 
-## 12.25.0b1 (Unreleased)
+## 12.24.1 (2025-01-21)
 
-### Features Added
+### Bugs Fixed
+- Fixed an issue where custom transports may encounter `AttributeError` on certain requests.
```

Comment:
Have we reviewed this before? I think this is fine as-is but I am wondering if there is any other possible changelog entries that are slightly less vague but still accurate. Will defer to 🧙

Reply:
I wrote that so personally, I think it's fine :)

Intentionally vague since it didn't seem to affect many people.